### DISCUSSION TITLE
Looking for convincing to contribute

### DISCUSSION THREAD

**Discussion by NathanielPather** [Imadnajam/Undermovies-app#32] [General]
- Is your application live? (link on github is dead)
- Are your services free or do users need to pay to watch stuff?
- What content do you offer?

I've never really felt the need to use a streaming service. I just want to be convinced as to why I should contribute. Currently my only reason is because I hate my current employer lol.

**Comment by Imadnajam** [comment]
> * Is your application live? (link on github is dead)
> * Are your services free or do users need to pay to watch stuff?
> * What content do you offer?
> 
> I've never really felt the need to use a streaming service. I just want to be convinced as to why I should contribute. Currently my only reason is because I hate my current employer lol.

Hello **NathanielPather**
`Are the services free or paid?`
free just we streem trailler  offer free content with ads

`What content is offered?`
The type of content offered depends on the streaming service. Most streaming platforms provide a variety of movies and TV series across different genres. Some might also have exclusive content or original productions.

`Reasons to contribute ?`

> Skill development: Improve your coding skills by working on a real-world project.
> Community: Contribute to a community of like-minded individuals who share an interest in movies and technology.
> Portfolio: Showcase your contributions on your resume or portfolio.
> Gratitude: If you enjoy the service, contributing is a way to express gratitude and help the project thrive.
